cron/check_parameters.php

```diff
@@ -3,7 +3,7 @@
   +----------------------------------------------------------------------+
   | PHP QA GCOV Website                                                  |
   +----------------------------------------------------------------------+
-  | Copyright (c) 2005-2018 The PHP Group                                |
+  | Copyright (c) 2005-2019 The PHP Group                                |
```

Comment:
```suggestion
  | Copyright (c) The PHP Group                                          |
```

These can be further simplified now because PHP headers don't include year ranges anymore... Thank you.

Reply:
This PR will be replaced to https://github.com/php/web-gcov/pull/9